tests/regression/stlexport/stl-import-limits-expected.stl

```diff
@@ -0,0 +1,86 @@
+solid OpenSCAD_Model
+  facet normal -1 0 0
+    outer loop
+      vertex 0 0 0
+      vertex 0 0 1.23456788
+      vertex 0 1.23456788 1.23456788
+    endloop
+  endfacet
+  facet normal 0 0 -1.47171605e-07
```

Comment:
Looks like this one still has the old values too. (leading zero on exponent, and some `-0` values further down in the file)